src/cascadia/TerminalApp/SettingsTab.cpp

```diff
@@ -50,7 +50,7 @@ namespace winrt::TerminalApp::implementation
     // - <none>
     // Return Value:
     // - The list of actions.
-    std::vector<ActionAndArgs> SettingsTab::BuildStartupActions() const
+    std::vector<ActionAndArgs> SettingsTab::BuildStartupActions(const bool /*asContent*/) const

```

Comment:
I'm guessing this is useful in the next PR?

Reply:
WHOOPS it should be in THIS pr

Reply:
WAIT no that is actually intentionally unused, because we just don't need it (other than just implementing the method from the base class).